### DISCUSSION TITLE
Mutiple skip index on one column in clickhouse,what's the execution logic?

### DISCUSSION THREAD

**Discussion by fiend19961** [ClickHouse/ClickHouse#57496] [Q&A]
-- create a test table
```sql
CREATE TABLE event_test
(
    `ts` DateTime64,
    `cuid` String,
    INDEX idx_cuid_mx cuid TYPE minmax GRANULARITY 10,
    INDEX idx_cuid_blm cuid TYPE bloom_filter GRANULARITY 100
)
ENGINE = MergeTree
PARTITION BY toYYYYMMDD(ts)
ORDER BY (ts)
SETTINGS index_granularity = 8192;
```
When I exec sql like this: 
```sql
select * from event_test where cuid = 'xxx'
``` 
What's the execution logic? 
idx_cuid_mx or idx_cuid_blm first,

Use minmax index first?
if minmax index not hit, then use bloom_filter. if minmax index hit , needn't use bloom_filter?

**Comment by lingtaolf** [comment]
![image](https://github.com/ClickHouse/ClickHouse/assets/87509571/a83593f1-aba6-4d17-a7d3-571d6cc9006a)
In terms of code, it will use all of your skipping indices to filter data.
Also, you can check it via explaining query .
![image](https://github.com/ClickHouse/ClickHouse/assets/87509571/a2b2cf0e-34fe-4365-ad41-1a0242b5199f)
